app/serializers/articles_show_serializer.rb

```diff
@@ -0,0 +1,14 @@
+class ArticlesShowSerializer < ActiveModel::Serializer
+  attributes :id, :title, :body, :date, :author
+
+  def author
+    {
+      first_name: object.user.first_name,
+      last_name: object.user.last_name
+    }
+  end
+
+  def date
+    object.created_at.strftime('%F')
```

Comment:
As I remember %F gives yy-mm-dd format. In principle there can be several articles per day, so it would be impossible to determine which is the freshest. maybe you should add hh-mm?

Reply:
Good point! Our controller handles the sorting regardless, but it will not be visible for the reader. This is now added!